languages/TazUnitTest.groovy

```diff
@@ -176,6 +178,18 @@ int currentBuildFileNumber = 1
 			println   "***  TAZ Unit Test job ${tazUnitTestRunJcl.submittedJobId} completed with $rc "
 			// Store Report in Workspace
 			new CopyToHFS().dataset(props.tazunittest_bzureportPDS).member(member).file(reportLogFile).copyMode(DBBConstants.CopyMode.valueOf("BINARY")).append(false).copy()
+			if (props.tazunittest_resultsConvtoJunit && props.tazunittest_resultsConvtoJunit.toBoolean()) {
+			// Convert the report to Junit and store in workspace
+			def exec = new UnixExec()
+			.command("Xalan")
+			.options(["-o", reportJunitFile.toString(), reportLogFile.toString(), xslFile])
+			.execute()
+		        if (exec != 0) {
+                             println "***  Warning: JUnit Conversion failed with return code RC=${exec}"
```

Comment:
Can you please have a look to the below linked snippet, how this is done in the other scripts to add a warning to the build result. 

https://github.com/IBM/dbb-zappbuild/blob/main/utilities/GitUtilities.groovy#L28C1-L30C43